### PR TITLE
feat(rest,types)!: User apps GA

### LINKED ISSUE #3695: [api-docs] Update User-Installed Apps Docs for GA (#6958)
A new commit was made into the api-docs repo: https://github.com/discord/discord-api-docs/commit/43170f8867ad0398dc00eeaf47093bda8ce85940
* update user install docs

* add 5 max followup callout

* add change log

* pass through application

* remove lingering preview labels

* update wording in tutorial

* update defaults

* api update

* wrong wording oop

* Update docs/Change_Log.md

Co-authored-by: vee <vendicated@riseup.net>

* Update docs/resources/Application.md

Co-authored-by: vee <vendicated@riseup.net>

---------

Co-authored-by: vee <vendicated@riseup.net>

This is a bot created issue.

### PR DESCRIPTION
**Pull request by Fleny113**
User applications are now GA (Generally Availably) and no longer in preview

> [!WARNING]
> This is breaking because `DiscordMessageInteractionMetadata.authorizing_integration_owners` was incorrectly typed to be using number keys instead of string as keys (which are numbers)

- Upstream: https://github.com/discord/discord-api-docs/pull/6958
- fixes #3695

### CODE REVIEW

File: packages/types/src/discordeno.ts [hunk 1]
```diff
@@ -466,9 +466,9 @@ export interface CreateSlashApplicationCommand {
   options?: Camelize<DiscordApplicationCommandOption[]>
   /** Set of permissions represented as a bit set */
   defaultMemberPermissions?: PermissionStrings[]
-  /** Installation context(s) where the command is available */
+  /** Integration types where the command is available */
```
Comment (lts20050703) [inline packages/types/src/discordeno.ts:469]:
```suggestion
  /**
   * Installation contexts where the command is available
   *
   * @remarks
   * This value is available only for globally-scoped commands
   * Defaults to the application configured contexts
   */
```

File: packages/types/src/discordeno.ts [hunk 2]
```diff
@@ -466,9 +466,9 @@ export interface CreateSlashApplicationCommand {
...
   integrationTypes?: DiscordApplicationIntegrationType[]
-  /** Interaction context(s) where the command can be used, only for globally-scoped commands. By default, all interaction context types included for new commands. */
+  /** Interaction context types where the command is available */
```
Comment (lts20050703) [inline packages/types/src/discordeno.ts:471]:
```suggestion
  /**
   * Interaction context(s) where the command can be used, only for globally-scoped commands.
   *
   * @remarks
   * By default, all interaction context types included for new commands.
   */
```

**Review by lts20050703 — COMMENTED** [review]
Feel free to ignore this review, I must say Discord makes it really confusing.

Like why are they referring to `integration_types` as [Installation contexts](https://discord.com/developers/docs/resources/application#installation-context) and `contexts` as [Interaction context(s)](https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-interaction-context-types) it's so confusing

File: packages/types/src/discord.ts [hunk 3]
```diff
@@ -2351,17 +2346,18 @@ export interface DiscordCreateApplicationCommand {
...
+   * Defaults to the application configured contexts
    */
   integration_types?: DiscordApplicationIntegrationType[]
   /**
-   * Interaction context(s) where the command can be used, only for globally-scoped commands. By default, all interaction context types included.
+   * Interaction context(s) where the command can be used, only for globally-scoped commands.
    *
    * @remarks
-   * This is currently in preview.
+   * By default, all interaction context types included for new commands.
```
Comment (lts20050703) [inline packages/types/src/discord.ts:2360]:
```suggestion
  /**
   * Interaction context(s) where the command can be used
   *
   * @remarks
   * This value is available only for globally-scoped commands
   * By default, all interaction context types included for new commands.
   */
```

**Comment by Fleny113** [comment]
I did not include the change of the wording for `contexts` and `integration_types` because if we change it then i feel like it is even more confusing

**Review by Skillz4Killz — APPROVED** [review]

**Review by AwesomeStickz — APPROVED** [review]

**Pull request merged**